app/scripts/controllers/reviewRegistration.js

```diff
@@ -68,20 +68,184 @@ angular.module('confRegistrationWebApp')
       return _.find($scope.blocks, {id: blockId});
     };
 
+    var setRegistrationAsCompleted = function() {
+      registration = angular.copy(registration);
+      registration.completed = true;
+
+      RegistrationCache.update('registrations/' + registration.id, registration, function () {
+        RegistrationCache.emptyCache();
+        if(conference.registrationCompleteRedirect){
+          $window.location.href = conference.registrationCompleteRedirect;
+        }else{
+          $route.reload();
+        }
+      }, function (data) {
+        $scope.currentRegistration.completed = false;
+        $scope.submittingRegistration = false;
+        modalMessage.error(data.error ? data.error.message : 'An error occurred while submitting your registration.');
+      });
+    };
+
+    ////// START EVENT-433 //////
+
+    // Get spouse registration
+    // End point: getSpouseRegistration = /conferences/conferenceId/registrations/spouse
+    ///// TEST DATA change end point url below to: /conferences/conferenceId/registrations/current /////
+    $http.get('conferences/' + conference.id + '/registrations/spouse')
+      .then(function(response) {
+        if (response.status === 200) {
+          $scope.spouseRegistration = response.data;
+          $scope.spouseExists = true;
+
+          ///// TEST DATA /////
+          // _.find($scope.spouseRegistration.registrants, 'email').email = 'janeworner3@hotmail.com';
+          // _.find($scope.spouseRegistration.registrants, 'firstName').firstName = 'Jane';
+          // _.find($scope.spouseRegistration.registrants, 'lastName').lastName = 'Worner3';
+          /////// TEST DATA /////
+
+        }
+        else {
+          $scope.spouseRegistration = {};
+          $scope.spouseExists = false;
+          console.log('No current and/or spouse registration found for user with ID.');
+        }
+      }).catch(function(response) {
+        $scope.spouseRegistration = {};
+        $scope.spouseExists = false;
+        console.log('No current and/or spouse registration found for user with ID. Status = ' + response.status);
+    })
+      .finally(function() {
+        checkSpouseRegistration();
+      });
+
+    //Parameters needed to retrieve all current registrants for conference.id
+    var queryParameters = {
+      block: [],
+      //page: 1,
+      //limit: 20,
+      orderBy: 'last_name',
+      order: 'ASC',
+      filter: '',
+      filterPayment: '',
+      filterRegType: '',
+      includeCheckedin: 'yes',
+      includeWithdrawn: 'yes',
+      includeIncomplete: 'yes'
+    };
+
+    //Get all current registrants based on conference.id and queryParameters
+    //then check if spouse has registered and display 'Spouse Registration' section if needed.
+    var checkSpouseRegistration = function() {
```

Comment:
Thank you Caleb.  In researching and looking at other code I actually found someone else using this same approach in another file.  I'm swamped today but hope to be able to work on this tomorrow and will give your short syntax a try.